packages/core-data/src/queried-data/reducer.js

```diff
@@ -232,7 +233,9 @@ const receiveQueries = compose( [
 	return {
 		itemIds: getMergedItemIds(
 			state?.itemIds || [],
-			action.items.map( ( item ) => item[ key ] ),
+			action.items
+				.map( ( item ) => item?.[ key ] )
+				.filter( ( itemId ) => itemId ),
```

Comment:
`filter( Boolean )` is yet another idiomatic way to do the same thing:

```suggestion
				.filter( Boolean ),
```

Reply:
Yeah, that's a nice shorthand. I went back and forth with filter's predicate, then decided to keep the current version.

Reply:
Up to you, it's not a big deal. Thanks 👍

Reply:
I had to rebase and included the update.